### ISSUE TITLE
address_format_id = 7 is for Australia

### ISSUE THREAD

**Issue by apmuthu** [zencart/zc-v1-series#538]
Line 2043 in ZenCart v1.5.4 of zc_install/sql/mysql_zencart.sql:
````
# 1 - Default, 2 - USA, 3 - Spain, 4 - Singapore, 5 - Germany, 6 - UK/GB
````
should be
````
# 1 - Default, 2 - USA, 3 - Spain, 4 - Singapore, 5 - Germany, 6 - UK/GB, 7 - Australia
````

**Comment by zcwilt** [comment]
Hi Thanks for the post. 
This has been fixed in v1.6.0
See https://github.com/zencart/zc-v1-series/commit/c128b7d3da3a5a0bd38ac7d0752d0ab0157bff7d

**Comment by scottcwilson** [comment]
I think this issue can be closed.
